tools/wasptool

```diff
@@ -258,6 +258,25 @@ def handle_binary_download(c, tname, fname):
     sz = eval(stat)
     bytes_read = 0
 
+    try:
+        if "/" in fname:  # create local directory
+            pile = "./"
+            for local_dir in fname.split("/")[:-1]:
+                if local_dir == ".":
+                    continue
+                try:
+                    os.mkdir(f"{pile}/{local_dir}/")
+                    print(f"Created local dir {pile}/{local_dir}/")
+                except FileExistsError:  # folder already exists
+                    pass
+                pile += local_dir + "/"
+        with open(fname, 'wb') as f:
+            f.write(b"write_test")
```

Comment:
Writing to the file is probably a bad idea. What if it contains important data? I'd just text for existence instead

Reply:
I felt the same way but followed logic of the original file : if you already had a fname file it would be overwritten.

If you still think it's better just tell me and I'll do it.

Reply:
I took your input into consideration and redid a new commit.

Reply:
I took your input into consideration and redid a new commit.